docs/methods/cardanoSignTransaction.md

```diff
@@ -17,31 +17,35 @@ TrezorConnect.cardanoSignTransaction(params).then(function(result) {
 
 ### Params 
 [****Optional common params****](commonParams.md)
-###### [flowtype](../../src/js/types/networks/cardano.js#L60-L179)
+###### [flowtype](../../src/js/types/networks/cardano.js#L87-L211)
 * `signingMode` - *obligatory* [CardanoTxSigningMode](#CardanoTxSigningMode)
-* `inputs` - *obligatory* `Array` of [CardanoInput](../../src/js/types/networks/cardano.js#L62)
-* `outputs` - *obligatory* `Array` of [CardanoOutput](../../src/js/types/networks/cardano.js#L77)
+* `inputs` - *obligatory* `Array` of [CardanoInput](../../src/js/types/networks/cardano.js#L87)
+* `outputs` - *obligatory* `Array` of [CardanoOutput](../../src/js/types/networks/cardano.js#L103)
 * `fee` - *obligatory* `String`
 * `protocolMagic` - *obligatory* `Integer` 764824073 for Mainnet, 42 for Testnet
 * `networkId` - *obligatory* `Integer` 1 for Mainnet, 0 for Testnet
 * `ttl` - *optional* `String`
 * `validityIntervalStart` - *optional* `String`
-* `certificates` - *optional* `Array` of [CardanoCertificate](../../src/js/types/networks/cardano.js#L124)
-* `withdrawals` - *optional* `Array` of [CardanoWithdrawal](../../src/js/types/networks/cardano.js#L131)
-* `auixiliaryData` - *optional* [CardanoAuxiliaryData](../../src/js/types/networks/cardano.js#143)
+* `certificates` - *optional* `Array` of [CardanoCertificate](../../src/js/types/networks/cardano.js#L150)
+* `withdrawals` - *optional* `Array` of [CardanoWithdrawal](../../src/js/types/networks/cardano.js#L158)
+* `auixiliaryData` - *optional* [CardanoAuxiliaryData](../../src/js/types/networks/cardano.js#173)
+* `mint` - *optional* [CardanoMint](../../src/js/types/networks/cardano.js#164)
+* `additionalWitnessRequests` - *optional* `Array` of `string | Array<number>` (paths). Used for multi-sig and token minting witness requests as those can not be determined from the transaction parameters.
 * `metadata` - *removed* - use `auxiliaryData` instead
 
 ### CardanoTxSigningMode
 
-[Type definition](../../src/js/types/trezor/protobuf.js#L78)
+[Type definition](../../src/js/types/trezor/protobuf.js#L98)
 
 #### `ORDINARY_TRANSACTION`
 
-Represents an ordinary user transaction transferring funds.
+Represents an ordinary user transaction transferring funds. Witnesses are determined from the transaction parameters and from `additionalWitnessPaths`.
 
 The transaction
 - *should* have valid `path` property on all `inputs`
 - *must not* contain a pool registration certificate
+- *must* contain only 1852 and 1855 paths
```

Comment:
An 1852 path can only be used for witness? Since that is outside of tx body, I find this a bit confusing.

Reply:
What about: "must contain only 1852 and 1855 witness requests"?

Reply:
https://github.com/vacuumlabs/connect/pull/14/commits/d611fc39786b9f4f912aaf2c95f2f3cfdcc422df